src/java.base/share/classes/java/nio/Direct-X-Buffer.java.template

```diff
@@ -78,18 +79,42 @@ class Direct$Type$Buffer$RW$$BO$
         }
 
         public void run() {
-            UNSAFE.freeMemory(address);
-            Bits.unreserveMemory(size, capacity);
+            try {
+                UNSAFE.freeMemory(address);
+                Bits.unreserveMemory(size, capacity);
+            } catch (Throwable x) {
+                // Old jdk.internal.ref.Cleaner behavior: when it fails, VM exits.
```

Comment:
I'd prefer not reference a class that no longer exists, instead just say that it preserves long standing behavior exit the VM. In the case of DBB, I can't think what exceptions/errors are possible here.

Reply:
Amended the comment.